CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Note: This CHANGELOG is only for the monitoring team to track all monitoring related changes. Please see OpenShift release notes for official changes.
 
+## 4.14
+
+- [#1931](https://github.com/openshift/cluster-monitoring-operator/pull/1931) add nodeExporter.collectors.ksmd settings.
```

Comment:
We agreed to stop using the CHANGELOG for everything that is already tracked in an epic.